glide.lock

```diff
@@ -1,15 +1,30 @@
-hash: fe11759f8785c35a54b84dcce121eab8c8f220a2c05ebe3f36087ebe5bf38528
-updated: 2016-06-22T13:31:27.2502096-07:00
+hash: e9be0ae2ba50b38578ec6b497b908924851ece4a9f796714bae46327e25b9ae9
+updated: 2016-08-31T11:13:52.9423908-07:00
 imports:
 - name: github.com/dgrijalva/jwt-go
   version: f0777076321ab64f6efc15a82d9d23b98539b943
   subpackages:
   - .
+- name: github.com/stretchr/testify
```

Comment:
From comments I thought `glide.yaml` was fine and we only wanted to keep `testImports` in lock file. I have updated `glide.yaml` file for testImports. 
```Go
...
  subpackages:
  - /pkcs12
testImports:
- package: github.com/stretchr/testify
  vcs: git
  repo: https://github.com/stretchr/testify.git
  subpackages:
  - /require
```

Take a look.